### DISCUSSION TITLE
Some questions about SLR

### DISCUSSION THREAD

**Discussion by LNTU-YS** [groops-devs/groops#273] [Q&A] [answered]
Hello,

I noticed that GROOPS has added an SLR module, and I tried it, but encountered problems. I don't know what to do, as shown in the following picture:

![image](https://github.com/user-attachments/assets/4a21d758-9808-49f3-8e5d-315722de4110)

Thank you in advance for your feedback and help. Another point to mention is: Can your team release an example of the basic configuration of SLR? I believe I can learn knowledge from it.

With Kind Regards
YUAN

**Comment by LNTU-YS** [comment]
Please allow me to supplement the configuration
![image](https://github.com/user-attachments/assets/e7ef2f9b-1ed4-4656-b398-3b33da70e105)

**Comment by tmayerguerr** [comment]
Hi YUAN,

For some reason, only one station with a single pass with 32 observations is used. It is impossible to determine 181 unknown parameters from 32 observations.

A cookbook for SLR processing is on our long TODO list, but I expect it could be a while yet.

regards
Torsten

**Reply by LNTU-YS** [reply]
Hello Torsten,

I have a new question now. My goal is to perform SLR residual verification, for which I only set “Parametrization-Troposphere”. But I found that the residual obtained by doing so is as high as several decimeters. Later, I found that adding “Parametrization-StaticPositions” would make the results normal, resulting in a result of about 2 centimeters.

Now, my question is, do I have to set “Parametrization-StaticPositions”? 

Or, can I set “Parametrization-StaticPositions” without setting “station static position parameters”? Because I found that when I set “Parametrization-StaticPositions”, the number of SLR data stations is less than 3, which makes it impossible to solve. This will result in many days of data that cannot yield results.

With Kind Regards
YUAN

**Reply by LNTU-YS** [reply]
Hello Torsten,

I have added “Parametrization-RangeBiasStationApriori”. Then I noticed that the input file from the “rangeBias” folder. But in the “rangeBias” folder, there are only 15 “rangeBias.xxxx.txt” files. I downloaded “ILRS_Data_Handling_File_xxxx.xx.xx.snx” and tried following “SlrSinexDataHandling2Files”, and found that I got the same data as the “rangeBias” folder. If the station file I need is not provided, how can I get it?

With Kind Regards
YUAN

**Comment by tmayerguerr** [comment, marked as answer]
Hi YUAN,

StaticPositions: You're right: in your case, you shouldn't estimate the positions. It is important that you introduce precise a priori coordinates with stations->inputfileStationPosition.

RangeBiasStationApriori: Most stations are calibrated and should not have a range bias. The few exceptions are contained in ILRS_Data_Handling_File_xxxx.xx.xx.snx. If your station is not included, the range bias is probably zero.

regards
Torsten

**Reply by LNTU-YS** [reply]
Hello Torsten,

Thank you very much for your reply and guidance, but I still have a question about “StaticPositions”. In fact, as you said, I had already set up “stations->inputfileStationPosition” before. But it still estimates the positions. The relevant situation is shown in the following figure.

![image](https://github.com/user-attachments/assets/137b419e-20d0-4710-92a4-220919c27564)

![image](https://github.com/user-attachments/assets/fe70538b-fbdd-4ccd-8503-19df9cab22b4)

It can be seen that even if I set “stations->inputfileStationPosition”, it will still estimate the positions. Then for many days, the number of SLR data stations was less than 3, which made it impossible to solve.

I don't have any good solutions for this and I hope to receive your help.

With Kind Regards
YUAN

**Reply by tmayerguerr** [reply]
If you want to use the a priori station coordinates (stations->inputfileStationPosition) without determining the coordinates, you must remove parametrization->staticPositions

**Reply by PangerDe** [reply]
Hi Yuan,
I am learning to use the SLR module. Could you please send me your configuration file for me to refer to. Thank you very much.
